src/CoreWCF.Primitives/src/CoreWCF/Description/MessagePropertyDescriptionCollection.cs

```diff
@@ -6,6 +6,6 @@ namespace CoreWCF.Description
     public class MessagePropertyDescriptionCollection : System.Collections.ObjectModel.KeyedCollection<string, MessagePropertyDescription>
     {
         internal MessagePropertyDescriptionCollection() { }
-        protected override string GetKeyForItem(MessagePropertyDescription item) { return default; }
+        protected override string GetKeyForItem(MessagePropertyDescription item) => item.Name;
```

Comment:
I don't know if relevant, but these other collections use `return default`
* `MessagePartDescriptionCollection`
* `MessageHeaderDescriptionCollection`
* `MessageDescriptionCollection`

Reply:
That does sound like a problem. I'll check against .NET Framework and make them match.